CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Please add your own contribution below inside the Master section
 Bug-fixes within the same version aren't needed
 
 ## Master
+* change *Run Related Tests* default keybinding to Ctrl-Alt/Option-T to avoid overriding default shortcuts.
```

Comment:
as a convention, please append the author name at the end (see previous change log)